src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,22 +94,65 @@ class WorkflowManagerActor(backend: Backend) extends Actor with CromwellActor {
     restartIncompleteWorkflows()
   }
 
+  private def reply[A](id: WorkflowId, fa: Future[A],
+                       onSuccess: (WorkflowId, A) => WorkflowManagerSuccessResponse,
+                       onFailure: (WorkflowId, Throwable) => WorkflowManagerFailureResponse): Unit = {
+    // Don't close over sender.
+    val sndr = sender
+    fa onComplete {
+      case Success(a) => sndr ! onSuccess(id, a)
+      case Failure(e) => sndr ! onFailure(id, e)
+    }
+  }
+
+  private def reply[A, B](id: WorkflowId, a: A, fb: Future[B],
+                          onSuccess: (WorkflowId, A, B) => WorkflowManagerSuccessResponse,
+                          onFailure: (WorkflowId, A, Throwable) => WorkflowManagerFailureResponse): Unit = {
+    // Don't close over sender.
+    val sndr = sender
```

Comment:
Minor: Some of these assignments are `val sndr = sender`, and others are `val sndr = sender()`.